test/libsolidity/semanticTests/viaYul/array_storage_length_access.sol

```diff
@@ -1,19 +1,19 @@
 contract C {
-	uint[] storageArray;
-	function set_get_length(uint256 len) public returns (uint256)
-	{
-		storageArray.length = len;
-		return storageArray.length;
-	}
-
+    uint[] storageArray;
+    function set_get_length(uint256 len) public returns (uint256)
+    {
+        storageArray = new uint[](len);
+        return storageArray.length;
+    }
 }
 // ====
-// compileViaYul: true
+// EVMVersion: >=petersburg
 // ----
 // set_get_length(uint256): 0 -> 0
 // set_get_length(uint256): 1 -> 1
 // set_get_length(uint256): 10 -> 10
 // set_get_length(uint256): 20 -> 20
 // set_get_length(uint256): 0 -> 0
 // set_get_length(uint256): 0xFF -> 0xFF
-// set_get_length(uint256): 0xFFFF -> 0xFFFF
+// set_get_length(uint256): 0xFFF -> 0xFFF
```

Comment:
Here it would have been nice to be able to run a test suite with different evm versions :)